bucket/psfzf.json

```
@@ -0,0 +1,20 @@
+{
+    "version": "2.4.0",
+    "description": "A PowerShell wrapper around the fuzzy finder fzf",
+    "homepage": "https://github.com/kelleyma49/PSFzf",
+    "license": "MIT",
+    "notes": "See usage: https://github.com/kelleyma49/PSFzf#usage",
```

```suggestion
    "license": "MIT",
```

Don't add usage notes in manifest.